src/boot/categories.rs

```diff
@@ -76,10 +77,10 @@ fn categories_from_toml(
     Ok(result)
 }
 
-pub fn sync_with_connection(toml_str: &str, conn: &mut impl Conn) -> Result<()> {
+pub async fn sync_with_connection(toml_str: &str, conn: &mut AsyncPgConnection) -> Result<()> {
     use crate::schema::categories;
     use diesel::pg::upsert::excluded;
-    use diesel::RunQueryDsl;
+    use diesel_async::RunQueryDsl;
```

Comment:
Should this be used globally within this file now?

Reply:
yep, makes sense. updated :)